src/python/WMCore/ReqMgr/DataStructs/RequestStatus.py

```diff
@@ -144,12 +144,16 @@
 # is name of the status
 REQUEST_STATE_LIST = REQUEST_STATE_TRANSITION.keys()
 
-ACTIVE_STATUS_FILTER = {"RequestStatus": ['assigned', 'staging', 'staged', 'acquired',
+ACTIVE_STATUS_FILTER = {"RequestStatus": ['assignment-approved', 'assigned', 'staging', 'staged', 'acquired',
                                           'running-open', 'running-closed', 'force-complete',
                                           'completed', 'closed-out']}
-ACTIVE_NO_CLOSEOUT_FILTER = {"RequestStatus": ['assigned', 'staging', 'staged', 'acquired',
+ACTIVE_NO_CLOSEOUT_FILTER = {"RequestStatus": ['assignment-approved', 'assigned', 'staging', 'staged', 'acquired',
                                                'running-open', 'running-closed', 'force-complete',
                                                'completed']}
+# used to determine active workflows requiring parent datasets
+ACTIVE_NO_CLOSEOUT_PARENT_FILTER = {"RequestStatus": ['assignment-approved', 'assigned', 'staging', 'staged', 'acquired',
+                                               'running-open', 'running-closed', 'force-complete',
+                                               'completed'], "IncludeParents": True}
```

Comment:
I think this `IncludeParents` filter doesn't belong to this module. We can get back to it in the future though